src/app/resources/iis_scripts/local_http.ps1

```diff
@@ -0,0 +1,59 @@
+param(
+    [string]
+    $requestBase64
+)
+
+$ErrorActionPreference = "Stop"
+$contentEncoding = [System.Text.Encoding]::UTF8
+
+function stringify($content) {
+    if ((!$content) -or ($content -is [System.String])) {

```

Comment:
Can we refactor this as the following in order to remove else?

```
    if ((!$content) -or ($content -is [System.String])) {
        return $content
    } 

    if ($content -is [System.Byte[]]) {
        return [System.Convert]::ToBase64String($content)
    } 
    return $content.ToString()
    }
```

Reply:
Ok